frappe/public/js/frappe/views/workspace/workspace.js

```diff
@@ -68,6 +68,7 @@ frappe.views.Workspace = class Workspace {
 		this.cached_pages = $.extend(true, {}, this.sidebar_pages);
 		this.all_pages = this.sidebar_pages.pages;
 		this.has_access = this.sidebar_pages.has_access;
+		this.has_create_access = this.sidebar_pages.has_create_access;
```

Comment:
Isn't this the same as doing `frappe.model.can_create("Workspace")`?

Reply:
From what i understood, `can_create` considers the "User Cannot Create" checkbox irrespective of the roles who have create permission and doesn't allow creation of private workspaces for any role.